bl/evaluation/printer.go

```diff
@@ -129,8 +129,9 @@ func parseToPrinterWarnings(results *EvaluationResults, invalidYamlFiles []*vali
 				rule := rules[ruleId]
 				details := printer.WarningInfo{
 					Caption:     rule.Name,
-					Occurrences: rule.Count,
+					Occurrences: rule.getCount(),
```

Comment:
Is the `getCount` available to you in this context? I guess not since it is not Capitalized

Reply:
somehow it still worked, weird

Reply:
ohhh, I only used it in the "evaluation" package, so it still worked.
fixed it now.